ldregistry/templates/about/simpleworkflow.vm

```diff
@@ -0,0 +1,10 @@
+<li>Go to GitHub (you will need an account, and to be logged in) and go to https://github.com/wmo-registers/code-manuals/tree/master/typo_fixes</li>
```

Comment:
please may this be a clickable link?

is it sensible to make it open in a new page as well, so that the workflow page is kept open?

```
<a href="http://www.quackit.com/html/tags/" target="_blank">HTML Tags</a>
```